src/components/MapView/PendingMapView.tsx

```diff
@@ -4,18 +4,22 @@ import {View} from 'react-native';
 import BlockingView from '@components/BlockingViews/BlockingView';
 import Icon from '@components/Icon';
 import * as Expensicons from '@components/Icon/Expensicons';
+import useTheme from '@hooks/useTheme';
 import useThemeStyles from '@hooks/useThemeStyles';
 import variables from '@styles/variables';
 import type {PendingMapViewProps} from './MapViewTypes';
 
-function PendingMapView({title = '', subtitle = '', style}: PendingMapViewProps) {
+function PendingMapView({title = '', subtitle = '', style, isSmallIcon = false}: PendingMapViewProps) {
     const hasTextContent = !_.isEmpty(title) || !_.isEmpty(subtitle);
     const styles = useThemeStyles();
+    const theme = useTheme();
+    const iconSize = isSmallIcon ? variables.iconSizeSuperLarge : variables.iconSizeUltraLarge;
```

Comment:
NAB: It would be more clear to pass the icon size directly as a prop. It's kind of funny to say that it's a small icon and the use a "super large" icon 😆

Reply:
Changed it to `isSmallerIcon`

Reply:
I was thinking pass `variables.iconSizeSuperLarge` as an `iconSize` prop directly, for example, but what you did is fine too